op-node/rollup/interop/standard/system.go

```diff
@@ -0,0 +1,69 @@
+package standard
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/event"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+)
+
+// StandardMode makes the op-node follow the canonical chain based on a read-only supervisor endpoint.
+type StandardMode struct {
+	log log.Logger
+
+	emitter event.Emitter
+
+	cl *sources.SupervisorClient
+
+	mu sync.RWMutex
+}
+
+func NewStandardMode(log log.Logger, cl *sources.SupervisorClient) *StandardMode {
+	return &StandardMode{
+		log:     log,
+		emitter: nil,
+		cl:      cl,
+	}
+}
+
+func (s *StandardMode) AttachEmitter(em event.Emitter) {
+	s.emitter = em
+}
+
+func (s *StandardMode) OnEvent(ev event.Event) bool {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	switch x := ev.(type) {
+	case rollup.ResetEvent:
+		s.log.Error("todo: interop needs to handle resets", x.Err)
+		// TODO(#13337): on reset: consolidate L2 against supervisor, then do force-reset
+	}
```

Comment:
I'm not sure why a Standard Mode Subsystem would need to do anything on Reset. It appears to be stateless